src/Controller/Api/GroupController.php

```diff
@@ -100,7 +100,7 @@ public function getGroupsAction(ParamFetcherInterface $paramFetcher)
      *
      * @ApiDoc(
      *  requirements={
-     *      {"name"="id", "dataType"="integer", "requirement"="\d+", "description"="group id"}
+     *      {"name"="id", "dataType"="string", "description"="group id"}
```

Comment:
```suggestion
     *      {"name"="id", "dataType"="string", "description"="Group identifier"}
```
Can we improve description too?